src/test/java/seedu/address/logic/commands/EventAddVolunteerCommandTest.java

```diff
@@ -64,6 +64,25 @@ public void execute_duplicateVolunteer_throwsCommandException() {
                 -> command.execute(model));
     }
 
+    @Test
+    public void execute_clashingEvent_throwsCommandException() {
+        // Assign a volunteer to an existing event
+        model.getEventStorage().getEventList().get(model.getEventStorage().getEventList().size() - 1)
+                .addVolunteer(model.getVolunteerStorage().getVolunteerList()
+                        .get(model.getVolunteerStorage().getVolunteerList().size() - 1));
+
+        // Assign the volunteer a clashing event
+        model.getVolunteerStorage().getVolunteerList().get(model.getVolunteerStorage().getVolunteerList().size() - 1)
+                .addEvent(model.getEventStorage().getEventList()
+                        .get(model.getEventStorage().getEventList().size() - 2));
```

Comment:
can be split into multiple variables. i think `model.getEventStorage().getEventList()` and `model.getVolunteerStorage().getVolunteerList()` can be split into variables since they are used multiple times

Reply:
facts